decryptor/mysql/column_field.go

```diff
@@ -103,11 +108,13 @@ func ParseResultField(data []byte) (*ColumnDescription, error) {
 		//length of default value lenenc-int
 		field.DefaultValueLength, _, n, err = LengthEncodedInt(data[pos:])
 		if err != nil {
+			log.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorProtocolProcessing).WithError(err).Errorln("can't get length encoded integer of default value length")
```

Comment:
I would change wording to:

"SQL parsing error: can't get length encoded integer of default value length"

Reply:
Here we parse packet of mysql protocol, not sql.